### PR TITLE
(maint) Make needed variables accessible in tasks via `let`

### PR DESCRIPTION
**Pull request by underscorgan**

### CODE REVIEW

File: spec/lib/packaging/artifactory_spec.rb [hunk 1]
```diff
@@ -9,31 +9,33 @@
...
+      },
+      'solaris-10-sparc' => {
         :artifact => "./solaris/10/PC1/puppet-agent-5.3.1.34.gf65f9ef-1.sparc.pkg.gz",
         :repo_config => '',
-    },
+      },
+    }
   }
 
   platform_tags = {
```
Comment (melissa) [inline spec/lib/packaging/artifactory_spec.rb]: can you also update `platform_tags`?

Comment (melissa) [reply spec/lib/packaging/artifactory_spec.rb]: or does that not work with rspec?

Comment (mwaggett) [reply spec/lib/packaging/artifactory_spec.rb]: +1

Comment (underscorgan) [reply spec/lib/packaging/artifactory_spec.rb]: that doesn't work with rspec, we're only accessing platform_tags outside of tests

**Comment by melissa** [comment]
I'm merging this, as it's a trivial change.

**Pull request merged**